docs/config.rst

```diff
@@ -41,12 +71,30 @@ Optional settings
 
         {
             dataCallback: function(data) {
-                // remove references to the environment
-                delete data.request.env;
```

Comment:
Since there is no more `request.env` / `process.env` ... updated this example to set a user.